### ISSUE TITLE
Adding `.Get` to `FuncMap`

### ISSUE THREAD

**Issue by Nosferican** [gohugoio/hugo#4561]
Similar to #3141.

**Comment by bep** [comment]
Explain?

**Comment by Nosferican** [comment]
```
{{ apply .Site.RegularPages "Get" "Name" }}
```
Return the names of all the regular pages.

**Comment by paulcmal** [comment]
What are you trying to achieve? Can't you simply loop over `.Site.RegularPages`?

Or if you really want it as a list of names and nothing else, I guess you could do something like this:
```
{{ $.Scratch.Delete "pages" }}
{{ range .Site.RegularPages }}
    {{ $.Scratch.Add "pages" .Name }}
{{ end }}
{{ $.Scratch.Get "pages" }}
```

**Comment by Nosferican** [comment]
I wanted to get the section urls in a site with the following structure:
```
- Content
- _index.md
--Blog
---A
----_index.md
----a.md
----b.md
---B
----_index.md
----a.md
----b.md
```
I wanted to have a widget in home `blog.html` layout to display the name and urls to [Blog/A and Blog/B], but since the sections don't have the urls I was getting the unique `.Dir` of pages under `Blog/` and trying to use those to build the section urls. I saw a similar problem [here](https://github.com/vjeantet/hugo-menu-show/). Noticed that `map` wasn't supporting a few functions and I saw the merge to add `print`. I thought `Get` would be pretty useful too especially for other parameters which could simplify the loops.

Pretty sure there is a much simpler solution to what I was trying to do, but regardless adding `Get` to `FuncMap` would be welcoming.
